tests/cli/pinfo_tool.py

```diff
@@ -116,31 +116,29 @@ def testPrintStorageInformationAsText(self):
         'binary_cookies', 'bsm_log', 'chrome_cache', 'chrome_preferences',
         'cups_ipp', 'custom_destinations', 'dockerjson', 'dpkg', 'esedb',
         'esedb/esedb_file_history', 'esedb/msie_webcache', 'filestat',
-        'firefox_cache', 'firefox_cache2', 'hachoir', 'java_idx', 'lnk',
-        'mac_appfirewall_log', 'mac_keychain', 'mac_securityd', 'mactime',
-        'macwifi', 'mcafee_protection', 'mft', 'msiecf', 'olecf',
-        'olecf/olecf_automatic_destinations', 'olecf/olecf_default',
-        'olecf/olecf_document_summary', 'olecf/olecf_summary', 'openxml',
-        'opera_global', 'opera_typed_history', 'pe', 'plist',
-        'plist/airport', 'plist/apple_id', 'plist/ipod_device',
+        'firefox_cache', 'firefox_cache2', 'hachoir',
```

Comment:
Please undo changes, seeing this file does not need to be changed in this CL.

Reply:
Done, if I found the right SHA hash for `git revert`.